var/spack/repos/builtin/packages/faodel/package.py

```diff
@@ -35,11 +35,17 @@ class Faodel(CMakePackage):
                          low-level communication')
 
     depends_on('mpi', when='+mpi')
-    depends_on('boost+mpi', when='+mpi', type='build')
-    depends_on('boost~mpi', when='~mpi', type='build')
+    depends_on('boost@1.58.0:')
+    depends_on('cmake@3.8.0:')
```

Comment:
My understanding is that if `type` is not specified, `depends_on` uses the default `type` value that is `build` and `link`.  Should `cmake` only be a `build` dependency?

Reply:
Yes it should.  I'll fix it.

Reply:
> My understanding is that if `type` is not specified, `depends_on` uses the default `type` value that is `build` and `link`.  Should `cmake` only be a `build` dependency?

Technically yes, although it doesn't actually matter since there's nothing in the cmake install prefix that would get picxked up by the link dependency, but you are correct: it should be `type='build'`